templates/download/server/manual.html

```diff
@@ -1,204 +1,405 @@
+          <input type="hidden" name="version" value="{{ releases.lts.full_version }}">
+          <input type="hidden" name="architecture" value="amd64">
+          <input type="hidden" name="next-step" value="download">
+          <button style="margin-right: 1rem;" type="submit" class="p-button--positive" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Server download', 'eventAction' : 'Select option', 'eventLabel' : '{{ releases.lts.full_version }}', 'eventValue' : undefined });">Download {{ releases.lts.full_version }} LTS</button>
+        </form>
+        <span class="u-text--muted">ISO image{% if releases.lts.iso_download_size %}, {{ releases.lts.iso_download_size }}{% endif %}</span>
+        <p><a href="#downloads">Alternative downloads&nbsp;&rsaquo;</a></p>
+        <p><a href="#architectures">Alternative architectures&nbsp;&rsaquo;</a></p>
+      </div>
+
+      <div class="js-tab-container">
+        <nav class="p-tabs">
+          <ul class="p-tabs__list js-tabbed-content u-no-margin--bottom" role="tablist" data-maintain-hash="true">
+            <li class="p-tabs__item">
+              <a class="p-tabs__link" role="tab" aria-selected="true" id="release-notes-tab" href="#release-notes" aria-controls="release-notes">Release notes</a>
+            </li>
+            <li class="p-tabs__item">
+              <a class="p-tabs__link" role="tab" aria-selected="false" id="system-requirements-tab" href="#system-requirements" aria-controls="system-requirements" tabindex="-1">System requirements</a>
+            </li>
+            <li class="p-tabs__item">
+              <a class="p-tabs__link" role="tab" aria-selected="false" id="how-to-install-tab" href="#how-to-install" aria-controls="how-to-install" tabindex="-1">How to install</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+
+      <div class="p-tabs__content p-section" id="release-notes" role="tabpanel" aria-labelledby="release-notes-tab">
+        <ul class="p-list--divided">
+          <li class="p-list__item is-ticked">OpenSSL 3.0 for modern, general-purpose cryptography and secure communication</li>
+          <li class="p-list__item is-ticked">Native host and guest drivers for NVIDIA virtual GPU (vGPU) software 14</li>
+          <li class="p-list__item is-ticked">Network acceleration improvements with SmartNIC support in Netplan</li>
+          <li class="p-list__item is-ticked">General support for GlusterFS, FRRouting and realmd/adcli under the Ubuntu main component</li>
+          <li class="p-list__item is-ticked">Linux 5.15 kernel for the recent hardware and security updates</li>
+          <li class="p-list__item is-ticked">Updates to QEMU, libvirt, PHP, Ruby, GCC, Python, MySQL, OpenLDAP, Samba</li>
+        </ul>
+        <hr class="p-rule" />
+        <p>
+          <a href="https://discourse.ubuntu.com/t/jammy-jellyfish-release-notes/24668">Full {{ releases.lts.full_version }} release notes&nbsp;&rsaquo;</a>
```

Comment:
This appears as '22.04' in the doc, should check which one is correct

Reply:
22.04 is correct, release notes refer to the major release